rows.go

```diff
@@ -80,24 +106,63 @@ func (rs *rowSets) empty() bool {
 	return true
 }
 
+// Column is a mocked column Metadate for rows.ColumnTypes()
+type Column struct {
```

Comment:
maybe in that case, instead of having exported construction methods, just export the struct fields instead?